src/api-reference/request/v4.endpoints.markdown

```diff
@@ -132,19 +135,19 @@ GET /v4/requests/{requestUuid}
 
 None
 
-### HTTP Response
+### HTTP response
 
 #### Payload
 
-[Request](#schema-request) - The Request having {requestUuid} as unique identifier
+[Request](#request) - The Request having {requestUuid} as unique identifier
```

Comment:
@charlieconcur ok. That's weird because I've copied the latest version from developer.concur.com and copied it from Github last week, and those #scheme-request where not displayed. Will make the changes